packages/circle-demo-webapp/app/components/WalletBalance/WalletBalance.tsx

```diff
@@ -22,7 +26,9 @@ export function WalletBalance({ balance }: WalletBalanceProps) {
         <p className="text-base font-medium text-gray-900">
           {balance.amount} {balance.token.symbol}
         </p>
-        <p className="text-sm text-gray-500">{balance.token.name}</p>
+        {isShowBlockchainName && (
```

Comment:
What’s the purpose of isShowBlockchainName? The name suggests it relates to the blockchain, but it actually displays token.name.

Also, this component has been updated in my branch based on client feedback, so it might be better to create a new component specifically for TokenSelect instead of modifying this one.